runtime/tutor

```diff
@@ -1340,6 +1340,182 @@ actions available in a popup. This will allow you to:
 
 
 
+=================================================================
+=                  CHAPTER 13.1 CREATE NEW SPLIT                =
+=================================================================
+
+In normal mode, press ctrl-w to open the Window menu, and display
+a list of available commands.
+
+To open a new empty buffer in a vertical split on the right half
+of your current window, you can use ctrl-w nv (i.e., press ctrl
+and w at the same time, then press n, then press v). Your current
+window will now split in 2 vertically, a new empty buffer split
+will appear on the right half, and your cursor will jump to the
+new vertical split.
+
+To create a new empty buffer in an horizontal split, press
+ctrl-w ns . This will split your current window in 2 horizontally
+and create a new buffer, and your cursor will jump to the new
+horizontal split.
+
+
+
+
+=================================================================
+=                    CHAPTER 13.2 MOVE BETWEEN SPLITS           =
+=================================================================
+
+Use ctrl-w k or ctrl-w arrow-up to move to the split above your
+current split. Use ctrl-w j or ctrl-w arrow-down to move back
+to the split below. Use ctrl-w h or ctrl-w arrow-left to move
+to the split left, and ctrl-w l or ctrl-w arrow-right to move
+back to the split right. To move to the next split (in opening
+order), press ctrl-w w .
```

Comment:
It seems unnecessary to repeat `Ctrl-w+k`, `Ctrl-w+j`, etc. Could this say something like "From the window minor mode (Ctrl+w), one can move around with hjkl or the arrow keys. For example,  `Ctrl+w l` moves to the split to the right. To move through splits in the order that they were opened, use `Ctrl-w w`."